recurrence/base.py

```diff
@@ -832,10 +840,7 @@ def serialize(rule_or_recurrence):
         A rfc2445 formatted unicode string.
     """
     def serialize_dt(dt):
-        if not dt.tzinfo:
-            dt = localtz().localize(dt)
-        dt = dt.astimezone(pytz.utc)
-
+        dt = to_utc(dt)
```

Comment:
Serialize supports easy naive-to-aware conversion and UTC normalization via the new `to_utc`